index.ts

```diff
@@ -9,30 +9,39 @@ async function connectDb() {
   try {
     await prisma.$connect();
     logger.info("Database connected");
-  } catch (err) {
-    logger.error("Error connectig to database: ", err);
+  } catch (error) {
+    logger.error("Error connecting to database: ", error);
   }
 }
 
 connectDb();
 
 app.get("/", async (req, res) => {
-  // prisma.user
-  //   .create({
-  //     data: {
-  //       firstName: "D",
-  //       lastName: "Blazanovic",
-  //     },
-  //   })
-  //   .then(() => {
-  //     res.end("user created");
-  //   })
-  //   .catch((err) => {
-  //     res.end(`Error: ${err}`);
-  //   });
-  const users = await prisma.user.findMany();
-  logger.info(users);
-  res.json(users).end();
+  try {
+    const users = await prisma.user.findMany();
+    logger.info(users);
+    res.json(users).end();
+  } catch (error) {
+    logger.error(error);
+    res.end("Error occurred: " + error);
+  }
+});
+
+app.get("/add", async (req, res) => {
```

Comment:
Since this is just a test endpoint can you delete it, and also /delete endpoint, in the tickets in TODO section you will have a chance to implement new endpoints :)

Reply:
I removed those two endpoints.